src/User32.Desktop/User32+VirtualKey.cs

```diff
@@ -0,0 +1,1062 @@
+﻿// Copyright (c) to owners found in https://github.com/AArnott/pinvoke/blob/master/COPYRIGHT.md. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+
+namespace PInvoke
+{
+    using System;
+
+    /// <content>
+    /// Contains the virtual-key code constants.
+    /// </content>
+    public partial class User32
+    {
+        /// <summary>
+        /// Virtual-key codes
+        /// Defined in winuser.h from Windows SDK v6.1
+        /// Documentation pulled from MSDN.
+        /// </summary>
+        public enum VirtualKey : int
```

Comment:
Thanks for all this including documentation!
You don't need to say "Documentation pulled from MSDN" though, especially in the `<summary>` where it shows up in Intellisense. If you want to keep it in `<remarks>` that would be OK, but almost all our xml docs come from MSDN, and we don't mention it everywhere else.